pyproject.toml

```diff
@@ -36,17 +36,13 @@ numpy = ">=1.17 < 1.18"
 nuscenes-devkit = ">=1.0.2 <1.0.3"
```

Comment:
Let's remove nuscene dataset here.

Reply:
FYI: nuscenes-devkit has dependencies/includes following components:
  - nuscenes and kitty dataset
  - scikit-learn and opencv-python